sample-apps/nodejs-java/k8s/service.yaml

```diff
@@ -38,7 +38,7 @@ spec:
     spec:
       containers:
         - name: javashowcase-service
-          image: %REGISTRY_LOCATION%-docker.pkg.dev/%GCLOUD_PROJECT%/%CONTAINER_REGISTRY%/java-sample-service:latest
+          image: "%REGISTRY_LOCATION%-docker.pkg.dev/%GCLOUD_PROJECT%/%CONTAINER_REGISTRY%/java-sample-service:latest"
```

Comment:
Curious: Did it work without the quotes ?

Reply:
Good question, I'm not sure how this was missed before. This sample merged a couple minutes before we added the CI check, so maybe it just snuck in. Will keep an eye on it